templates/website/stats-mp-performance.html

```diff
@@ -8,6 +8,11 @@
 $sort = $info['sort'];
 $mp = $info['mp'];
 
+$pc = $values['pc'];
+if ($mp) {
```

Comment:
As you're passing the postcode in separately now, is the `mp['pc']` setting here (and in stats.php) needed at all any more?